cfgov/v1/models/sublanding_filterable_page.py

```diff
@@ -50,9 +51,49 @@ def get_context(self, request, *args, **kwargs):
         return filterable_context.get_context(self, request, context)
 
     def get_form_class(self):
-        return filterable_context.get_form_class()
+        return forms.FilterableListForm
 
     def get_page_set(self, form, hostname):
         return filterable_context.get_page_set(self, form, hostname)
-     
 
+
+class ActivityLogPage(SublandingFilterablePage):
+    template = 'activity-log/index.html'
+
+    def get_page_set(page, form, hostname):
+        results = {}
+        selections = {}
+
+        # Get filter selections for Blog and Report
+        for f in page.content:
+            if 'filter_controls' in f.block_type and f.value['categories']['page_type'] == 'activity-log':
+                categories = form.cleaned_data.get('categories', [])
+                selections = {'blog': False, 'research-reports': False}
+                for category in selections.keys():
+                    if not categories or category in categories:
+                        selections[category] = True
+                        if category in categories:
+                            del categories[categories.index(category)]
+        # Get Newsroom pages
+        if categories:
+            try:
+                parent = CFGOVPage.objects.get(slug='newsroom')
+                results['newsroom'] = AbstractFilterPage.objects.live_shared(hostname).descendant_of(parent).filter(form.generate_query())
+            except CFGOVPage.DoesNotExist:
+                print 'Newsroom does not exist'
```

Comment:
> guaranteed in the future we'll need to add validation for the fields on the wagtail admin.
 yup

> raised exception, so we'll be email notified.
As per [my comment](https://github.com/cfpb/cfgov-refresh/pull/1922#discussion_r60246792), being notified by email makes sense, but again I don't want to prevent the whole page from being viewable just because one source of content is dead. Is there a better way?